### ISSUE TITLE
generated spec tests don't pass rubocop

### ISSUE THREAD

**Issue by igalic** [nwops/puppet-retrospec#36]
our (@puppet-community's) [.rubocopy.yml](https://github.com/puppet-community/modulesync_config/blob/master/moduleroot/.rubocop.yml) is rather generous) but it still won't pass the autogenerated spec tests from puppet-retrospec

one of the most glaring issues seems to be the way `.with` is split (the dot should be on the next line.
as well as the preference of `""`, over `''`.

**Comment by logicminds** [comment]
You can always fork and modify the templates here: https://github.com/nwops/retrospec-templates.  Furthermore, I think it would be a great addition to have the rubocop autogenerated by retrospec then ensure the templates conform to rubocop's rules.

**Comment by logicminds** [comment]
Also, you can add your fork to the config file: https://github.com/nwops/puppet-retrospec#config-file in your home directory.

**Comment by logicminds** [comment]
@igalic I have improved some formatting with the use of awesome print.  However this will require a new dependency.  The output is below. I tried to convert the double quotes to single quotes by redefining the String.#inspect method but the output was worse so I am just relying on String.#tr for now. However, I fear this method may end up replacing escaped `"` where we don't want it to.  I suppose its worth it though.   Since this requires a new dependency I won't be able to include this template in the main repo until I release 0.11.   

```ruby
it do
    is_expected.to contain_concat__fragment('XXreplace_meXX-filters')
      .with(
        'content' => 'template(apache/vhost/_filters.erb)',
        'order'   => '330',
        'target'  => '$priority_realregsubst($name,  , _, G).conf'
      )
  end
